csharp-algorithms/FindAllPositionsOfAString/FindAllPositionsOfAString/Algorithms/FirstAttempt.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace FindAllPositionsOfAString.Algorithms;
+
+public class FirstAttempt
+{
+    public static List<int> FindAll(string text, string searchText)
+    {
+        List<int> positions = [];
+
+        var index = text.IndexOf(searchText);
```

Comment:
Just a note here, but this is performing a search using CurrentCulture for matching. If that is expected it is fine, but we may want to still explicitly list that for clarity sake. This also applies in the while loop.

Reply:
Stated in the article